src/Core/Board.flix

```diff
@@ -46,17 +46,19 @@ namespace Flixball/Core/Board {
     }
 
 
-    pub def players(b: Board): List[(PlayerId, Position)] = {
+    pub def players(b: Board): Map[PlayerId, Position] = {
         let Board(br) = b;
         br.tiles |> Map.toList |> List.filterMap(match (coords, t) -> match t {
             case Person(id, dir) => Some(id, Position(coords, dir))
             case _ => None
-        })
+        }) |> List.toMap
     }
 
     pub def player(b: Board, id: PlayerId): Option[Position] =
-        players(b) |>
-            List.find(match (foundId, _) -> foundId == id) |>
-            Option.map(match (_, pos) -> pos)
+        let Board(br) = b;
```

Comment:
Why don't we use `players` anymore?

Reply:
For efficiency, I generally think functions with internal access should use internal data. But it probably doesn't matter too much here